multimodal/src/autogluon/multimodal/utils.py

```diff
@@ -1768,6 +1775,8 @@ def extract_from_output(outputs: List[Dict], ret_type: str, as_ndarray: Optional
             ret[feature_name] = torch.cat([ele[feature_name] for ele in feature_masks])
     elif ret_type == BBOX:
         return [ele[BBOX] for ele in outputs]
+    elif isinstance(ret_type, List):
```

Comment:
A list is an invalid type to extract information from output. `ret_type` should be some key in the return of model. For the text recognition, we may call `extract_from_output ` separately with `text` and `score`:
`
ret = []
for per_ret_type in ["text", "score"]:
    per_ret = extract_from_output(outputs=outputs, ret_type=per_ret_type)
    ret.append(per_ret)
`

Reply:
Done.